src/routes/Search.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+import { PharmacyList } from "../containers/PharmacyList";
+
+interface SearchRouteParam {
+  postalCode: string;
+}
+
+// eslint-disable-next-line import/no-default-export
```

Comment:
Why are you disabling this? We should either follow the rule or we disable it. My issue with default exports is they can be named arbitrarily while imported. Named exports must be imported without ambiguity.

Reply:
React lazy requires default exports.

> React.lazy takes a function that must call a dynamic import(). This must return a Promise which resolves to a module with a default export containing a React component.

Reply:
dang it... i keep forgetting about that